CorsixTH/Lua/languages/russian.lua

```diff
@@ -2610,16 +2610,16 @@ humanoid_name_ends = {
   [26] = "атич",
 }
 vip_names = {
-  [1] = "Мэр Нижнего Магнитошахтинска",
-  [2] = "Лоуренс Найтингейл",
-  [3] = "Бернард, король Нидерландов",
-  [4] = "Онг Сян Су Куй, предводитель бирманской демократической оппозиции",
-  [5] = "Сэр Риджинальд Крамбли",
-  [6] = "Сэр Вилли Сэвил",
-  [7] = "Советник Кроуфорд Первс",
-  [8] = "«Ракета» Ронни Джепсон",
-  [9] = "Футболист премьер-лиги",
-  [10] = "Л. Ф. Пробст III",
+  [1]  = "Мэр Нижнего Магнитошахтинска",
+  [2]  = "Леонид Каневский",
+  [3]  = '«Всезнающий» Онотоле Вассерман',
```

Comment:
@Matroftt is there a reason why this isn't using `"` instead?

Reply:
Used " symbols in name at first, but then switched to », and forgot to revert. My bad

Reply:
Coolio, @MandrakeHorse you cool to update the quotation marks?

Reply:
Had a bit of trouble with Git, but the single quotes have been replaced with double quotes